datalab/datalab.sh

```diff
@@ -130,6 +130,8 @@ function run_datalab(){
 }
 
 function main(){
+  install_docker
```

Comment:
You could make this not happen if docker is installed so that users can provide their own docker init action.